### ISSUE TITLE
undefined symbol in typed_ast._ast3 module when running on 3.9.8

### ISSUE THREAD

**Issue by anthrotype** [google/pytype#1044]
using pytype 2021.11.2 with Python 3.9.8 on Linux gives the following error:

```
ImportError: lib/python3.9/site-packages/typed_ast/_ast3.cpython-39-x86_64-linux-gnu.so: undefined symbol: _PyUnicode_DecodeUnicodeEscape
```

See full log: https://github.com/googlefonts/nanoemoji/runs/4167299470?check_suite_focus=true#step:5:40

The upstream issue seems to be https://github.com/python/typed_ast/issues/169

It works if I run under 3.9.7.

**Comment by lgeiger** [comment]
Starting with Python 3.8 `typed-ast` recommends to use the stdlib `ast` module as it supports al features of `typed-ast`, are there any plans to switch to the upstream `ast` module for newer versions of Python?

**Comment by rchen152** [comment]
Sorry for the slow response! It looks like typed_ast is going to get a new release in which this issue is fixed (https://github.com/python/typed_ast/issues/169#issuecomment-965658010), so once that lands and we update the typed_ast version we depend on, that should fix the immediate issue.

Long-term, @martindemello has volunteered to look into switching pytype to use ast instead of typed_ast in newer versions.

**Comment by rchen152** [comment]
Opened https://github.com/google/pytype/issues/1045 for switching from typed-ast to ast.

**Comment by rchen152** [comment]
I prepared a PR requiring typed-ast 1.5.0. All the pytype tests passed on it, so you should be able to unbreak pytype by manually installing typed-ast 1.5.0. I should be able to push out a new pytype release tomorrow.

**Comment by rchen152** [comment]
Alright, this should be fixed in pytype-2021.11.12.

**Comment by anthrotype** [comment]
Thanks!
